### PR TITLE
Masterdiet 59 - startForm view

### PR DESCRIPTION
**Pull request by RafisSomeone**

### CODE REVIEW

File: src/start/StartForm.js [hunk 1]
```diff
@@ -27,15 +45,27 @@ class StartForm extends Component {
...
+      active: 'BMR',
+      activeChanged: false,
+      goalChanged: false
     }
 
     this.handleSubmit = this.handleSubmit.bind(this)
   }
 
+  handleSelectedActivity = selection => {
+    if (selection === this.state.active) {return}
```
Comment (jakubowiczish) [inline src/start/StartForm.js:57]: You could reformat this file

File: src/start/GoalBox.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+class GoalBox extends Component {
+  constructor (props) {
+    super(props)
+  }
+
+  parseHeader = header => {
+    return header.charAt(0).toUpperCase() + header.slice(1).toLowerCase().replace('_', ' ')
+  }
+
+
```
Comment (jakubowiczish) [inline src/start/GoalBox.js:40]: 1 newline is enough

**Review by jakubowiczish — APPROVED** [review]

File: src/start/StartForm.js (hunk 1 above)
Comment (RafisSomeone) [reply src/start/StartForm.js:57]: formatted

File: src/start/GoalBox.js (hunk 2 above)
Comment (RafisSomeone) [reply src/start/GoalBox.js]: deleted

**Pull request merged**